modules/task/README.md

```diff
@@ -0,0 +1,49 @@
+# Async task module
+
+The async task module presents an distributed task execution framework
+for services that want to execute a function asynchronously and in a durable
+fashion. It achieves durability with "backends" that are messaging transports.
+To use the module, initialize it at service startup and register any functions
+that will be invoked async. Simply call task.Enqueue to call a function async
+and the execution framework will take care of running it at a later time.
```

Comment:
The backend transport. I was trying to avoid too much detail here but you're right, people will wonder that. Will clarify.